app/js/controllers/admin/testimonial.js

```diff
@@ -0,0 +1,111 @@
+const AdminTestimonialCtrl = function ($scope, $state, $mdDialog, $mdToast, AdminDataService, ClientDataService, $sce) {
+
+    $scope.testimonials = getTestimonials();
+    $scope.selected = [];
```

Comment:
Not sure where this $scope.variable.selected is being used, I haven't seen it being used anywhere within the controller and view.

Reply:
Done. Made the change to remove the unused variable